src/DrevOps/BehatScreenshotExtension/Context/ScreenshotAwareContextInterface.php

```
@@ -30,10 +30,12 @@ interface ScreenshotAwareContextInterface extends Context {
    *   File name pattern failed.
    * @param array<int,string> $info_types
    *   Show these info types in the screenshot.
+   * @param array<string,mixed> $animation
+   *   Animated GIF settings (keys: enabled, frame_delay).
    *
    * @return $this
    */
-  public function setScreenshotParameters(string $dir, bool $on_failed, string $failed_prefix, bool $always_fullscreen, bool $on_every_step, string $filename_pattern, string $filename_pattern_failed, array $info_types): static;
+  public function setScreenshotParameters(string $dir, bool $on_failed, string $failed_prefix, bool $always_fullscreen, bool $on_every_step, string $filename_pattern, string $filename_pattern_failed, array $info_types, array $animation): static;
```

PHPMD (2.15.0)

[error] The parameter $on_failed is not named in camelCase. (undefined)

(CamelCaseParameterName)